src/components/Profile/MyPosts/Post/Post.js

```diff
@@ -0,0 +1,17 @@
+import React, { Component } from 'react';
+
+import s from './Post.module.css';
+
+const Post = (props) => {
+    return (
+        <div className={s.item}>
+            <img src='https://yt3.ggpht.com/a/AGF-l79zTH00bozogFe1oAjQyY7-lnxI-uZMKgonrw=s900-mo-c-c0xffffffff-rj-k-no'></img>
```

Comment:
Consider storing image url in a separate constant.

Reply:
In the future I plan to use the object for avatars, now it's just a workpiece